.env

```diff
@@ -98,7 +98,6 @@ CKANEXT___GOOGLEANALYTICS__IDS=google-analytics-fake-key-testing-87654321
 # The maximum content size, in bytes, for uploads
 CKAN__STORAGE__MAX_CONTENT_LENGTH=650000000
 CKAN_STORAGE_PATH=/var/lib/ckan/files
-CKANEXT__S3FILESTORE__AWS_BUCKET_NAME
```

Comment:
Why was this line deleted?

Reply:
It was throwing an error for me during my local testing. The full string is the line below: `CKANEXT__S3FILESTORE__AWS_BUCKET_NAME=inventory`. I'm not sure why it wasn't causing an issue in other cases, to be honest.

Reply:
Interesting.. okay.  Yeah, that makes sense 👍